packages/components/stories/toggle-switch/toggle-switch.stories.js

```diff
@@ -26,7 +26,7 @@ stories
                         background: 'var(--general-section-1)',
                     }}
                     >
-                        <Switch is_enabled={is_enabled} handleToggle={() => setEnabled(!is_enabled)} />
+                        <Switch id="toggle-switch-story" is_enabled={is_enabled} handleToggle={() => setEnabled(!is_enabled)} />
```

Comment:
why is this needed?

Reply:
@negarn The component that @mustofa-binary wrote for `Switch` in the deriv-components needs a required `id` prop to wrok properly. He forgot to add `id` in the storybook so it was not working tproperly in the storybook. I have just fixed it

Reply:
I see, could you use single quotations?